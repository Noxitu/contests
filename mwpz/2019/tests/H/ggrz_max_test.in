3
0 831143535290875990
0 831143535290875991
0 831143535290875992
