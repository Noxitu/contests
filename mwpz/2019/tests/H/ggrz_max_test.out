999999999999999990
1000000000000000000
NO